lib/namespace

```diff
@@ -24,6 +24,11 @@ bind -a '#r' /dev
 mount -a /srv/slashn /n
 mount -a /boot /bin
 
+# uroot
+tmpfs /bin/uroot &
```

Comment:
is tmpfs a directive the namespace file knows? 
or should this be `import tmpfs`

Reply:
tmpfs is the program tmpfs